clang/test/dpct/module_wrapper_gen.cu

```diff
@@ -0,0 +1,30 @@
+// RUN: dpct --extra-arg="--ptx" \
+// RUN:      --out-root=%T/module_wrapper_gen \
+// RUN:      --cuda-include-path="%cuda-path/include" %s
+// RUN: FileCheck %s --input-file=%T/module_wrapper_gen/module_wrapper_gen.dp.cpp
+
+// START
+
+__device__ float2 operator+(float2 a, float2 b) {
+  return float2{a.x - b.x, a.y - b.y};
+}
+
+extern "C" __device__ void externCNonKernel() {}
+
+__device__ void deviceFun() {}
+
+__global__ void nonExternCKernel() {}
```

Comment:
move to line:23, or it is not checked.

Reply:
I'm not sure that affects the behavior of the test -- the test should pass if there is exactly one `_wrapper` between the `// START` and `// END` comments, and moving this to line 23 should not affect that.